src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1817,6 +1817,10 @@ Additional information: {1}</value>
     <value>Default (Open Documents)</value>
     <comment>This text is a menu command</comment>
   </data>
+  <data name="Default_None" xml:space="preserve">
+    <value>Default (None)</value>
+    <comment>This text is a menu command</comment>
```

Comment:
really?  our default is 'None'?  that seems... extremely strange to me.  Wouldn't our default be better as 'Active File'?

Reply:
No, `None` is not really the default. However, if the user sets the global option value in Tools Options to `None`, then the context menu command in Solution explorer will show `Default (None)` to indicate that the fallback value is `None`:

![image](https://user-images.githubusercontent.com/10605811/150133458-4b6cd69b-56d6-47f4-9ff3-6a1bb2984eea.png)